### PR TITLE
Update install steps with descriptor name

### PR DESCRIPTION
**Pull request by tylerphelan**
- Add enable_automatic_dependency_updates flag
- Update 1.3 version to 1.4 in relevent places

### CODE REVIEW

File: getting-started.html.md.erb [hunk 1]
```diff
@@ -92,6 +92,8 @@ ytt -f /tmp/bundle/values.yaml \
 	-v pull_from_kp_default_repo=true \
```
Comment (tomkennedy513) [inline getting-started.html.md.erb:92]: this also needs to be --data-value-yaml now

File: installing.html.md.erb [hunk 2]
```diff
@@ -140,12 +140,14 @@ ytt -f /tmp/bundle/values.yaml \
 	-v pull_from_kp_default_repo=true \
```
Comment (tomkennedy513) [inline installing.html.md.erb:140]: --data-value-yaml

File: installing.html.md.erb [hunk 3]
```diff
@@ -187,6 +191,8 @@ ytt -f /tmp/bundle/values.yaml \
     -v pull_from_kp_default_repo=true \
```
Comment (tomkennedy513) [inline installing.html.md.erb:191]:
```suggestion
    --data-value-yaml pull_from_kp_default_repo=true \
```

**Review by dumez-k — APPROVED** [review]

File: getting-started.html.md.erb (hunk 1 above)
Comment (tomkennedy513) [reply getting-started.html.md.erb:92]:
```suggestion
	--data-value-yaml pull_from_kp_default_repo=true \
```

File: installing.html.md.erb [hunk 4]
```diff
@@ -129,7 +129,7 @@ Where `TBS-VERSION` and `IMAGE-REPOSITORY` are the same values used during reloc
 
 2. Use the [Carvel](https://carvel.dev/) tools `kapp`, `ytt`, and `kbld` to install Build Service and define the required Build Service parameters:
 
-Tanzu Build Service 1.3 ships with a dependency updater that can update ClusterStacks, ClusterStores, ClusterBuilders, and the CNB Lifecycle from TanzuNet automatically. Enabling this feature will keep Images up to date with the latest security patches and fixes, and is *highly recommended*. To enable this feature, pass in your TanzuNet credentials when running the install command below:
+Tanzu Build Service ships with a dependency updater that can update ClusterStacks, ClusterStores, ClusterBuilders, and the CNB Lifecycle from TanzuNet automatically. Enabling this feature will keep Images up to date with the latest security patches and fixes, and is *highly recommended*. To enable this feature, include your TanzuNet credentials, `descriptor_name`, and `enable_automatic_dependency_updates` when running the install command below:
```
Comment (tomkennedy513) [inline installing.html.md.erb:132]: should we mention that by default, if you just pass in tanzunet creds , tbs will bootstrap your cluster with deps, but not keep them up to date

File: getting-started.html.md.erb [hunk 5]
```diff
@@ -92,6 +92,8 @@ ytt -f /tmp/bundle/values.yaml \
...
 	-v tanzunet_username='tanzunet-username' \
 	-v tanzunet_password='tanzunet-password' \
+	-v descriptor_name='lite' \
+	--data-value-yaml enable_automatic_dependency_updates=true \
```
Comment (matthewmcnew) [inline getting-started.html.md.erb:96]: Should we provide context on if/when to use the automatic_updater?

File: installing.html.md.erb [hunk 6]
```diff
@@ -42,15 +42,15 @@ Build Service utilizes `PersistentVolumeClaims` to cache build artifacts, which
...
-	* These docs assume kp cli v0.4.* from TBS release v1.3.*. If a feature is not working, you may need to upgrade your cli.
+	* These docs assume kp cli v0.4.* from TBS release v1.4.*. If a feature is not working, you may need to upgrade your cli.
 
 * Download the `docker` CLI to authenticate with registries.
 
 * Download the Dependency Descriptor file (`descriptor-<version>.yaml`) from the latest release on the [Tanzu Build Service Dependencies](https://network.tanzu.vmware.com/products/tbs-dependencies/) page on Tanzu Network. This file contains paths to images that contain dependency resources Tanzu Build Service needs to execute image builds.
 
-<p class="note"><strong>Note:</strong> Clusters running with Containerd 1.4.1 are not compatible with TBS. Notably, TKG 1.2.1 uses this version of Containerd, a different TKG version must be used.</p>
+<p class="note"><strong>Note:</strong> Clusters running with Containerd 1.4.1, 1.5.6, and 1.5.7 are not compatible with TBS. Notably, TKG 1.2.1 and 1.5.0 use these versions of Containerd, a different TKG version must be used.</p>
```
Comment (matthewmcnew) [inline installing.html.md.erb:51]: TKG 1.5 has not been released to the public yet. I feel like would should wait and see if this is an issue?

File: getting-started.html.md.erb (hunk 1 above)
Comment (tylerphelan) [reply getting-started.html.md.erb:92]: we might have missed this var: https://github.com/pivotal/build-service/blob/main/deployments/tbs/config/schema.yaml#L7

Comment (tomkennedy513) [reply getting-started.html.md.erb:92]: :(

Comment (tomkennedy513) [reply getting-started.html.md.erb:92]: That's my bad

File: getting-started.html.md.erb (hunk 5 above)
Comment (tylerphelan) [reply getting-started.html.md.erb:96]: We say: 

```
Tanzu Build Service ships with a dependency updater that can update ClusterStacks, ClusterStores, ClusterBuilders, and the CNB Lifecycle from TanzuNet automatically. Enabling this feature will keep Images up to date with the latest security patches and fixes, and is *highly recommended*. To enable this feature, include your TanzuNet credentials, `descriptor_name`, and `enable_automatic_dependency_updates` when running the install command below:
```

which I still think is true.

File: getting-started.html.md.erb (hunk 1 above)
Comment (tylerphelan) [reply getting-started.html.md.erb:92]: I reviewed it so it's my fault

File: getting-started.html.md.erb (hunk 5 above)
Comment (tylerphelan) [reply getting-started.html.md.erb:95]: This is how I addressed some of these concerns along with @tomkennedy513 's

File: faq.html.md.erb [hunk 7]
```diff
@@ -320,3 +320,31 @@ By default, `imgpkg` will not relocate the proprietary windows layers to your re
 ## <a id='faq-21'></a> What is the relationship between a kpack *image resource* and an *OCI image*?
 
 "Image resource" describes a kubernetes custom resource that produces OCI images by way of build resources. This resource will continue producing new builds that, when successful, output container images to the registry as configured by the image resource.
+
+## <a id='faq-22'></a> Pinning the Tanzu Net Updater
+
+When you configure the Tanzu Network Auto Updater during installation, dependencies are pulled in from network.vmware.com as they are released to keep the all app OCI images up-to-date and patched.
```
Comment (tomkennedy513) [inline faq.html.md.erb:326]:
```suggestion
When you configure the Tanzu Network Auto Updater during installation, dependencies are pulled in from network.tanzu.vmware.com as they are released to keep the all app OCI images up-to-date and patched.
```

**Review by tomkennedy513 — APPROVED** [review]

**Review by dumez-k — APPROVED** [review]

File: getting-started.html.md.erb [hunk 8]
```diff
@@ -69,29 +69,30 @@ docker login registry.tanzu.vmware.com
 3. Relocate the images with the [Carvel](https://carvel.dev/) tool `imgpkg` by running:
 
 ```
-imgpkg copy -b "registry.tanzu.vmware.com/build-service/bundle:1.3.0" --to-repo my.registry.io/some-repo/tbs --registry-ca-cert-path /tmp/ca.crt
+imgpkg copy -b "registry.tanzu.vmware.com/build-service/bundle:1.4.1" --to-repo my.registry.io/some-repo/tbs --registry-ca-cert-path /tmp/ca.crt
```
Comment (matthewmcnew) [inline getting-started.html.md.erb:72]: Should this be 1.4.2?

File: installing.html.md.erb [hunk 9]
```diff
@@ -129,31 +129,31 @@ Where `TBS-VERSION` and `IMAGE-REPOSITORY` are the same values used during reloc
...
 	-v tanzunet_username='<TANZUNET-USERNAME>' \
 	-v tanzunet_password='<TANZUNET-PASSWORD>' \
+	-v descriptor_name='<DESCRIPTOR-NAME>' \
+	--data-value-yaml enable_automatic_dependency_updates=true \
 	| kbld -f /tmp/bundle/.imgpkg/images.yml -f- \
 	| kapp deploy -a tanzu-build-service -f- -y
 ```
 You can check the status of the DependencyUpdater by running `kubectl -n build-service get TanzuNetDependencyUpdater dependency-updater -o yaml`
 
-Alternatively, if you prefer to manage dependencies yourself (*not recommended*), simply leave the TanzuNet credentials out of the install.
```
Comment (matthewmcnew) [inline installing.html.md.erb:148]: I realize this is already in the docs but, is not recommended too strong?

**Review by matthewmcnew — APPROVED** [review]

**Pull request merged**